app/models/duty_calculator.rb

```diff
@@ -57,4 +57,18 @@ def additional_duty_rows
         acc << option_klass.new(measure, user_session, []).option
       end
   end
+
+  def default_options
+    return [waiver_option] if user_session.gb_to_ni_route?
+
+    []
+  end
+
+  def waiver_option
+    {}.tap do |option|
+      option[:key] = DutyOptions::Waiver.id
+      option[:evaluation] = DutyOptions::Waiver.new(nil, user_session, []).option
+      option[:priority] = DutyOptions::Waiver::PRIORITY
+    end
```

Comment:
Why not just: 

```ruby
{
   key: DutyOptions::Waiver.id,
  evaluation: DutyOptions::Waiver.new(nil, user_session, []).option,
 ...
}
```

Any reason why we need to use tap?

Reply:
So no good reason beyond I always wanted to return the option.